frontend/src/components/GardenctlConfigExample.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div>
+    <div class="text-h6 grey--text text--darken-1 my-4">Configuration</div>
+    The following is a sample configuration for <span class="font-family-monospace pl-1">gardenlogin</span> as well as <span class="font-family-monospace pl-1">gardenctl</span>.
+    Place the file under <code>~/.garden/gardenctl-v2.yaml</code>.
+    <code-block lang="yaml" :content="gardenctlConfigYaml" :show-copy-button="true"></code-block>
+    <div class="mt-2">
+      Alternatively, you can run the following <span class="font-family-monospace pl-1">gardenctl</span> command:
+    </div>
+     <code-block lang="shell" :content="configCmd" :show-copy-button="true"></code-block>
+    <div class="mt-2">
+      <v-icon class="pb-1" small>mdi-information-outline</v-icon>
+      Note that the
+      <span class="font-family-monospace">kubeconfig</span>
+      refers to the path of the garden cluster
+      <span class="font-family-monospace">kubeconfig</span>
+      which you can download from the <router-link :to="accountRoute">Account</router-link> page.
```

Comment:
Link does only work from me when I open in new tab, I get an endless spinner otherwise

Reply:
✅